src/App.tsx

```diff
@@ -1,26 +1,168 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { dataTodos, USER_ID } from './api/todos';
+import { Filter, Todo } from './types/Todo';
+import { Header } from './Components/Header';
+import { TodoList } from './Components/TodoList';
+import { Footer } from './Components/Footer';
+import { ErrorMessage } from './Components/ErrorMessage';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [todoTitle, setTodoTitle] = useState<string>('');
+  const [filter, setFilter] = useState(Filter.All);
+  const [loadingAllTodos, setLoadingAllTodos] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoadingAllTodos, setLoadingAllTodos] = useState<boolean>(false);
```
boolean should start with `is ` or `has`